### ISSUE TITLE
Development has stalled | Are there any good alternatives?

### ISSUE THREAD

**Issue by rubenvitt** [1kc/razer-macos#825]
I think, the development of this project has stalled since 2022-09, but there are so many issues & pull requests open.

Are there any good current alternatives to razer-macos that focus specifically on customizing Razer hardware on macOS? [OpenRGB](https://github.com/CalcProgrammer1/OpenRGB) supports Razer but is more of a general RGB lighting manager and does not work well for me (my Razer Basilisk v3 Pro freezes while the app is open). A native macOS app that deals with Razer or input-hardware specifically would be amazing.

I would love if someone continues developing razer-macos (whIch I'm not aware of). Otherwise, pointers to alternatives are very welcome. Thanks!

**Comment by nikhilsharma81181** [comment]
Use "OpenRGB" app for RGB customization and "SteerMouse 5" for button customization both app are available for mac,
I hope it helps for non-programmer/normal-users

**Comment by TheOneTheyCall-Tim** [comment]
Karabiner-Elements is unbeatable for button remapping, but doesn't do anything for dpi. That's what I'm here for. Love my Naga Epic, but it's too slow to respond after the death of Synapse.

**Comment by stickoking** [comment]
I don't think this repo is being maintained anymore I have made a fork to continue it's development recently added support for BlackWidow v4 75% and made a release 0.4.11 https://github.com/stickoking/razer-macos
